src/modules/units/util.ts

```diff
@@ -15,6 +15,10 @@ export const evictUnitsQuery = (projectId: string, unitGroupId?: string) => {
     id: `Project:${projectId}`,
     fieldName: 'unitGroups',
   });
+  cache.evict({
```

Comment:
Added this in order to make the capacity table appear right away when you add the first units to a group from the project units page.